src/subscription_manager/gui/widgets.py

```diff
@@ -114,7 +114,7 @@ def __init__(self):
         The initial_widget_names is a list of widgets to pull in as instance
         variables.
         """
-        self.log = logging.getLogger('rhsm-app.' + __name__ +
+        self.log = logging.getLogger(__name__ +
```

Comment:
Join here please  and every where else where the getLogger call can comfortably be all on one line.